GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GitMoveRenameTests.cs

```diff
@@ -139,7 +145,7 @@ public void GitStatusAfterUnstage()
         public void GitStatusAfterFileDelete()
         {
             string existingFilename = "test.cs";
-            this.Enlistment.GetVirtualPathTo(existingFilename).ShouldBeAFile(this.fileSystem);
+            this.EnsureTestFileExists(existingFilename);

```

Comment:
Why did you need to change the behavior here?

Reply:
Unfortunately all of the tests in this class were written to be completely dependent on *all* of the tests running in order.

`GitStatusAfterFileDelete` was assuming that all prior tests ran, and one of those tests creates "test.cs".  Because we can't run all of the tests just yet there is currently no "test.cs" at the point this test runs.

With this change you can run this individual test without having to run the tests listed above it.